src/data/faq.json

```diff
@@ -329,6 +341,7 @@
                         "textblock": [
                             "Starting with version 2.13, <b>vaccination, recovery, or test certificates</b> that you have deleted are temporarily moved to a recycle bin.  This allows you to easily recover the certificate in case of accidental removal and move it back to its original location. If a certificate remains in the recycle bin for 30 days, it will automatically be permanently deleted, unless you have manually permanently deleted it from the recycle bin beforehand. A certificate that has been deleted in the app can still be rescanned. If it is still in the recycle bin and has therefore not been permanently removed, you will receive a message that it has been restored.",
                             "Starting with version 2.14, <b>tests</b> that you have deleted will also be moved to the recycle bin for 30 days.",
+                            "Starting with version 2.17, the time at which an item in the recycle bin is finally deleted is displayed below the respective item.",

```

Comment:
"is finally deleted" =>

"will be finally deleted"

Reply:
Fixed in the latest commit, thanks!